graphql/resolvers.go

```diff
@@ -3951,25 +3940,21 @@ func (r *versionResolver) Status(ctx context.Context, obj *restModel.APIVersion)
 		if len(p.ChildPatches) > 0 {
 			patchStatuses := []string{*p.Status}
 			for _, cp := range p.ChildPatches {
-				patchStatuses = append(patchStatuses, *cp.Status)
-				// add the child patch tasks to tasks so that we can consider their status
-				childPatchTasks, _, err := task.GetTasksByVersion(*cp.Id, opts)
+				v, err := model.VersionFindOneId(*cp.Id)
```

Comment:
Can you please test this on staging to make sure that an aborted child patch updates the parent status correctly?

Reply:
I saw in staging that when task is aborted, the version that it belongs to will be set to failed and the aborted field will be set to true. The aborted field of the patch containing the child tasks will not be set. This resolver will calculate the status of the parent task (the one containing the child patches) based on all of the child patches statuses.

Reply:
@malikchaya2 Could you help me verify this condition: https://github.com/evergreen-ci/evergreen/pull/5554/files#diff-71a3e44f5234030321d7134c8dc226478f2d9f6f7349af9125cf5d595aedcf68R1081

Reply:
replying on slack